lib/core/versions/latest/Did.ts

```diff
@@ -63,24 +65,23 @@ export default class Did {
    */
   public static async create (didString: string, didMethodName: string): Promise<Did> {
     const methodNameParts = didMethodName.split(':');
-    if (methodNameParts.length < 2) {
+    if (methodNameParts.length > 2) {
```

Comment:
Thought we want to remove this check?

Reply:
Alright, I'm getting rid off it... I got rid of it anyways in the original change.